src/System.Drawing.Common/src/System/Drawing/Graphics.Windows.cs

```diff
@@ -417,85 +355,62 @@ public InterpolationMode InterpolationMode
 
-                int status = SafeNativeMethods.Gdip.GdipSetInterpolationMode(new HandleRef(this, NativeGraphics), unchecked((int)value));
-                SafeNativeMethods.Gdip.CheckStatus(status);
+                SafeNativeMethods.Gdip.CheckStatus(SafeNativeMethods.Gdip.GdipSetInterpolationMode(new HandleRef(this, NativeGraphics), (int)value));
             }
         }
 
         public GraphicsUnit PageUnit
         {
             get
             {
-                int unit = 0;
-                int status = SafeNativeMethods.Gdip.GdipGetPageUnit(new HandleRef(this, NativeGraphics), out unit);
-                SafeNativeMethods.Gdip.CheckStatus(status);
-
+                SafeNativeMethods.Gdip.CheckStatus(SafeNativeMethods.Gdip.GdipGetPageUnit(new HandleRef(this, NativeGraphics), out int unit));
                 return (GraphicsUnit)unit;
             }
             set
             {
                 if (value < GraphicsUnit.World || value > GraphicsUnit.Millimeter)
-                {
-                    throw new InvalidEnumArgumentException(nameof(value), unchecked((int)value), typeof(GraphicsUnit));
-                }
+                    throw new InvalidEnumArgumentException(nameof(value), (int)value, typeof(GraphicsUnit));
 
-                int status = SafeNativeMethods.Gdip.GdipSetPageUnit(new HandleRef(this, NativeGraphics), unchecked((int)value));
-                SafeNativeMethods.Gdip.CheckStatus(status);
+                SafeNativeMethods.Gdip.CheckStatus(SafeNativeMethods.Gdip.GdipSetPageUnit(new HandleRef(this, NativeGraphics), (int)value));
             }
         }
 
         public float PageScale
         {
             get
             {
-                var scale = new float[] { 0.0f };
-                int status = SafeNativeMethods.Gdip.GdipGetPageScale(new HandleRef(this, NativeGraphics), scale);
-                SafeNativeMethods.Gdip.CheckStatus(status);
-
-                return scale[0];
+                SafeNativeMethods.Gdip.CheckStatus(SafeNativeMethods.Gdip.GdipGetPageScale(new HandleRef(this, NativeGraphics), out float scale));
+                return scale;
             }
-
             set
             {
-                int status = SafeNativeMethods.Gdip.GdipSetPageScale(new HandleRef(this, NativeGraphics), value);
-                SafeNativeMethods.Gdip.CheckStatus(status);
+                SafeNativeMethods.Gdip.CheckStatus(SafeNativeMethods.Gdip.GdipSetPageScale(new HandleRef(this, NativeGraphics), value));
             }
         }
 
         public float DpiX
         {
             get
             {
-                var dpi = new float[] { 0.0f };
-                int status = SafeNativeMethods.Gdip.GdipGetDpiX(new HandleRef(this, NativeGraphics), dpi);
-                SafeNativeMethods.Gdip.CheckStatus(status);
-
-                return dpi[0];
+                SafeNativeMethods.Gdip.CheckStatus(SafeNativeMethods.Gdip.GdipGetDpiX(new HandleRef(this, NativeGraphics), out float dpi));
```

Comment:
Long code lines don't wrap well in the default github editor, would you mind breaking them up in one of theupcoming PRs?

Reply:
@Tanya-Solyanik you might try https://userstyles.org/styles/108591/github-wide. It is great for reading code/fixed format text on github.